transactionoutbox-core/src/main/java/com/gruelbox/transactionoutbox/DefaultPersistor.java

```diff
@@ -56,6 +56,12 @@ public class DefaultPersistor implements Persistor, Validatable {
   @SuppressWarnings("JavaDoc")
   private final Dialect dialect;
 
+  /**
+   * @param sequenceGenerator The sequence generator used for ordered tasks. Required
+   */
+  @SuppressWarnings("JavaDoc")
+  private final SequenceGenerator sequenceGenerator;
```

Comment:
On its own, this is a compatibility break; it requires all existing users to specify `DefaultSequenceGenerator` when creating a `DefaultPersistor`.  I suggest changing this class to have a constructor annotated with `@Builder`; then you can accept `null` for `sequenceGenerator` and replace it with a `DefaultSequenceGenerator`.

Reply:
fixed